test/user.test.js

```diff
@@ -254,27 +254,31 @@ describe('User', function() {
     });
 
     it('invalidates the user\'s accessToken when the user is deleted all', function(done) {
-      var usersId, accessTokenId;
+      var usersId = [];
+      var accessTokenId;
       async.series([
         function(next) {
-          User.create([{ name: 'myname', email: 'b@c.com', password: 'bar' },
-          { name: 'myname', email: 'd@c.com', password: 'bar' }], function(err, user) {
-            usersId = user.id;
+          User.create([
+          { name: 'myname', email: 'b@c.com', password: 'bar' },
+          { name: 'myname', email: 'd@c.com', password: 'bar' },
+          ], function(err, users) {
+            usersId[0] = users[0].id;
+            usersId[1] = users[1].id;
```

Comment:
I had this function but I removed for the same of a simple array of length of 2. 
I think this is better in case we need to add more users for testing.